src/translation-assistant.js

```diff
@@ -379,7 +388,25 @@ const THOUSAND_SEPARATOR_AS_THIN_SPACE_LOCALES = ['cs', 'fr', 'de',
  */
 function translateMath(math, lang) {
 
+    const placeholder = 'THSEP';
```

Comment:
Any reason not to extract these to file level as constants?

Reply:
Hmm, this placeholder is used only in this function (at least at this point) so I don't think it needs to be accessible outside. Am I missing something?

Reply:
Maybe I am misunderstanding what do you mean by "these". I'm definitely gonna extra locale lists from `mathTranslations`